### PR TITLE
Revise sktwocnt codebase

### PR DESCRIPTION
**Pull request by vanderhe**
Fixes unwanted formatting introduced by PR #17. While I was familiarizing myself with the codebase, I slightly expanded the documentation of the modules/functions/variables for the purpose of my own understanding; it is also included in the PR.

### CODE REVIEW

File: sktwocnt/lib/bisection.f90 [hunk 1]
```diff
@@ -1,36 +1,49 @@
...
+    !> value to locate ind for
+    real(dp), intent(in) :: x0
+
+    !> located element such that xx(ind) < x0 < xx(ind)
     integer, intent(out) :: ind
+
+    !> optional, user-specified tolerance for comparisons
     real(dp), intent(in), optional :: tol
 
+    !> length of array to search
```
Comment (awvwgk) [inline sktwocnt/lib/bisection.f90:36]: Don't add docmarkers for variables not part of the function signature. Ford will lump them together with the some of the other docstrings and create incorrect API docs.

Comment (vanderhe) [reply sktwocnt/lib/bisection.f90:36]: I see, I wasn't aware of that. I can change the docmarkers to ordinary comments.

Comment (vanderhe) [reply sktwocnt/lib/bisection.f90:36]: What about declarations on the module level (before 'contains'), like some of the tolerance/length specifications ('!!' or '!>')?

Comment (awvwgk) [reply sktwocnt/lib/bisection.f90]: Those variables are `private` / `protected` and are captured by Ford with their docstring, but not necessarily displayed unless enabled in the project file.

Comment (vanderhe) [reply sktwocnt/lib/bisection.f90]: Alright, then the current state is fine I guess.

File: sktwocnt/lib/coordtrans.f90 [hunk 2]
```diff
@@ -146,54 +204,63 @@ subroutine coordtrans_ahlrichs2(c11, spheric, jacobi)
...
     spheric(1) = rr
     spheric(2) = acos(c11(2))
     spheric(3) = pi * (c11(3) + 1.0_dp)
 
-    jacobi = (zeta * (1.0_dp + c11(1))**alpha / log(2.0_dp)) &
-        &* (alpha * log(2.0_dp / (1.0_dp - c11(1))) / (1.0_dp + c11(1)) &
-        &+ 1.0_dp / (1.0_dp - c11(1))) * rr * rr * pi
+    jacobi = (zeta * (1.0_dp + c11(1))**alpha / log(2.0_dp)) * (alpha&
+        & * log(2.0_dp / (1.0_dp - c11(1))) / (1.0_dp + c11(1)) + 1.0_dp / (1.0_dp - c11(1)))&
+        & * rr * rr * pi
```
Comment (aradi) [inline sktwocnt/lib/coordtrans.f90:214]: Just a note: You don't have to use the 100-char line length in mathematical expressions, but can also break the line earlier, if it makes the expression easier to read. Whether the original or the newer is more readable, is of course a matter of taste :wink:

File: sktwocnt/lib/gridgenerator.f90 [hunk 3]
```diff
@@ -1,119 +1,154 @@
...
-        use common_accuracy, only: dp
-        real(dp), intent(in) :: oldc(:)
-        real(dp), intent(out) :: newc(:)
-        real(dp), intent(out) :: jacobi
-      end subroutine coordtrans
-    end interface
-    real(dp), allocatable, intent(out) :: grid(:, :)
-    real(dp), allocatable, intent(out) :: weights(:)
 
+  !> ???
```
Comment (aradi) [inline sktwocnt/lib/gridgenerator.f90:17]:
```suggestion
  !> Generates a 1D (radial) grid around two centers
```

File: sktwocnt/lib/gridgenerator.f90 [hunk 4]
```diff
@@ -1,119 +1,154 @@
...
-        real(dp), intent(out) :: jacobi
-      end subroutine coordtrans
-      function partition(r1, r2, dist, params)
-        use common_accuracy, only: dp
-        real(dp), intent(in) :: r1, r2, dist, params(:)
-        real(dp) :: partition
-      end function partition
-    end interface
+
+  !> ???
```
Comment (aradi) [inline sktwocnt/lib/gridgenerator.f90:63]: Generates a 
```suggestion
  !> Generates a 2D (radial and azimuthal) grid around two centers.
```

File: sktwocnt/lib/gridgenerator.f90 [hunk 5]
```diff
@@ -1,119 +1,154 @@
...
+        r2a = sqrt(rtmpa - rtmpb) ! dist > 0
+        r2b = sqrt(rtmpa + rtmpb) ! dist < 0
+
+        rtmpa = - 0.5_dp * (dist**2 + r2a**2 - r1**2) / (dist * r2a)
+        rtmpb = 0.5_dp * (dist**2 + r2b**2 - r1**2) / (dist * r2b)
+
+        ! make sure, we are not sliding out from [-1,1] range for acos
         rtmpa = min(rtmpa, 1.0_dp)
-        rtmpa = max(rtmpa, -1.0_dp)
+        rtmpa = max(rtmpa, - 1.0_dp)
```
Comment (aradi) [inline sktwocnt/lib/gridgenerator.f90:129]:
```suggestion
        rtmpa = max(rtmpa, -1.0_dp)
```
(No space between unary `-` and number)

File: sktwocnt/lib/gridgenerator.f90 [hunk 6]
```diff
@@ -1,119 +1,154 @@
...
         rtmpb = min(rtmpb, 1.0_dp)
-        rtmpb = max(rtmpb, -1.0_dp)
+        rtmpb = max(rtmpb, - 1.0_dp)
```
Comment (aradi) [inline sktwocnt/lib/gridgenerator.f90:131]:
```suggestion
        rtmpb = max(rtmpb, -1.0_dp)
```

**Review by aradi — COMMENTED** [review]
I'm fine with the PR, had just found a few cosmetic issues.

**Comment by vanderhe** [comment]
Thanks for the feedback, I applied your suggestions (but probably not at every point of the codebase).

**Pull request merged**